cMake/UseLibPack3.cmake

```diff
@@ -0,0 +1,75 @@
+set(ENV{PATH} "${FREECAD_LIBPACK_DIR};$ENV{PATH}")
+set(ENV{CMAKE_PREFIX_PATH} ${FREECAD_LIBPACK_DIR})
```

Comment:
Why is this modifying CMAKE_PREFIX_PATH as an env?

Reply:
you probably want `list(PREPEND CMAKE_PREFIX_PATH "${FREECAD_LIBPACK_DIR}")` (or `APPEND` if you want people to take control)

Reply:
Copied from the old LibPack -- I've modified as you suggested.